### ISSUE TITLE
Travis test failing on master

### ISSUE THREAD

**Issue by supersammy7000** [laurent-clouet/sheepit-client#185]
I saw the Travis test was failing on the master branch and I looked into it and I can't figure out why. I forked the branch and right away mine was successful. If you look into the logs there was no commit between the successful build #202 and failing #203. The only change was the test version so I replicated the exact version on my branch build #3.3 and it still successfully built.

The error spit out from the failed job is no "ant" installed however it is required in our config and previously was working.

**Comment by MCOfficer** [comment]
The latest successful build was made using trusty, while the latest ran on xenial. So it's most likely because the default build image switched from trusty to xenial ([which doesn't have ant by default](https://docs.travis-ci.com/user/reference/xenial/#jvm-clojure-groovy-java-scala-support)).

either
```yaml
before_install:
  - sudo apt-get update
  - sudo apt-get install -y libxml2-dev
```

or 

```yaml
dist: trusty
```

should fix this problem.

**Comment by supersammy7000** [comment]
The apt-get didn't work however forcing trusty as was the previous default did fix the issue. I created #186 which implements the change.

**Comment by MCOfficer** [comment]
whoops, that's what i get for pasting from the travis docs. i forgot the change the apt package. now it should work.

**Comment by supersammy7000** [comment]
@laurent-clouet fixed it in commit 6b71f9f
